osbs/build/spec.py

```diff
@@ -354,6 +357,23 @@ def set_params(self, git_uri=None, git_ref=None,
         self.yum_proxy.value = yum_proxy
         self.koji_parent_build.value = koji_parent_build
 
+        odcs_enabled = True if odcs_url and odcs_openidc_secret else False
+        if (signing_intent or compose_ids) and not odcs_enabled:
```

Comment:
All of these validations are not covered by unit tests.

Reply:
they are already